test/gentlemanjerry.bats

```diff
@@ -149,3 +178,28 @@ teardown() {
   run timeout 10 java -cp /tmp/test SslTest api.logentries.com 25414
   [ "$status" -eq 0 ]
 }
+
+@test "Gentleman Jerry can connect to a TLSv1.2-only Endpoint" {
+  wait_for_tls12_server
+  run timeout 5 openssl s_client -CAfile /tmp/certs/jerry.crt -connect localhost:4433
+  [[ "$output" =~ "Verify return code: 0 (ok)" ]]
```

Comment:
See below as well — I'm not a huge fan of using `run` then checking `output` if you could just use `... | grep`, although this is not a big deal.